docs/guides/transfers.md

```diff
@@ -32,17 +32,31 @@ Coming soon
 
 ### Cross chain transfers
 
-?> At present, we only support transferring KSM from Kusama to Kintsugi
+?> At present we support transferring KSM between Kusama and Kintsugi
+
+#### Kusama -> Kintsugi
 
 1. Go to the Transfer page on the app. Make sure you allow polkadot.js to connect to the website.
 2. Select the Cross Chain Transfer tab.
 3. Make sure you select the correct account from the account selector at the top of the page. This is the account from which the transfer will be made.
-4. Your available balance on the relay chain is shown above the amount field. This wil be different from the parachain balance shown at the top of the page.
-5. The "from" chain value will automatically be set to Kusama and the "to" chain value will be set to Kintsugi.
-6. Select an account from the destination select field. You can only transfer to accounts that are in your wallet.
+4. Your available balance on the relay chain (Kusama) is shown above the amount field. This wil be different from the parachain balance shown at the top of the page.
+5. Set the "from" chain value to Kusama and the "to" chain value to Kintsugi. You should only need to set one field, as the other field will be changed automatically.
+6. Select an account from the destination select field. You can only transfer to accounts that are in your wallet.If you want to transfer using the same account, make sure that the destination account matches the account selector at the to of the page.
 7. Click "Transfer" and sign the extrinsic to make a transaction.
 8. After the transfer has been made, if you select the account/wallet that has received the KSM from the account selector then the top right KSM balance should be updated to include the amount of KSM transferred.
 
+#### Kusama -> Kintsugi
```

Comment:
@tomjeatt this should be Kintsugi to Kusama, right? And all the fields below should say from Kintsugi and To Kusama?

Reply:
Yes :) should be fixed now